src/TensorFlow.jl

```diff
@@ -165,14 +165,13 @@ library loaded.
 macro py_proc(expr)
     quote
         eval(Main, quote
-            remotecall_fetch($(TensorFlow.load_python_process())) do
+            remotecall_wait($(TensorFlow.load_python_process())) do
```

Comment:
It's because Python loads its own version of libtensorflow in the main namespace, which conflicts with TensorFlow.jl's use of libtensorflow. It crashed when I tried and it seems to be a known issue. It would be great to find a solution that didn't involve using a second namespace.